code401challenges/array-reverse/ArrayReverse.java

```diff
@@ -0,0 +1,27 @@
+public class ArrayReverse {
+    public static void main(String args[]) {
+        int[] array1 = {1, 3, 4, 6, 2, 5, 10, 9, 20};
+        int[] reverseArray = reverseArray(array1);
+        for (int el : reverseArray){
+            System.out.println(el);
+        }
+    }
+    public static int[] reverseArray(int[] array) {
+        int halfway;
+        if (array.length % 2 == 0) {
+            halfway = array.length/2;
+        } else {
+            halfway = (array.length + 1)/2;
+        }
+        for(int i = 0; i < halfway; i++){
+            if (i == halfway && array.length % 2 != 0){
+                return array;
+            }
+            int temp;
+            temp = array[i];
+            array[i] = array[array.length - 1 - i];
+            array[array.length - 1 - i] = temp;
+        }
+        return array;
+    }
+}
```

Comment:
This angry red symbol is telling you that you don't have an empty newline at the end of the file--GitHub wants there to be a newline at the end of every file so it can display these diffs more effectively. Not a big deal, you're not losing points for it, just want to let you know what it means!